sections/char.zsh

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env zsh
```

Comment:
Add shebang to all other sections as well, or we will forget this when adding new tests

Reply:
That was just an attempt to make Travis green 😄

Actually, we don't need shebangs, so I've dropped them.